tests/unit/plugins/modules/test_nmcli.py

```diff
@@ -3947,6 +4077,8 @@ def test_bond_connection_unchanged(mocked_generic_connection_diff_check, capfd):
                              choices=['broadcast', 'roundrobin', 'activebackup', 'loadbalance', 'lacp']),
             # team active-backup runner specific options
             runner_hwaddr_policy=dict(type='str', choices=['same_all', 'by_active', 'only_active']),
+            # team lacp runner specific options
+            runner_fast_rate=dict(type='bool', default=False),
```

Comment:
```suggestion
            runner_fast_rate=dict(type='bool'),
```
Argspec in nmcli.py differs by absent of 'default' value. Probably forgot to change it.